src/Tools/dotnet-monitor/FormatterOptions.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel;
+
+#if UNITTEST
+namespace Microsoft.Diagnostics.Monitoring.UnitTests.Options
+#else
+namespace Microsoft.Diagnostics.Tools.Monitor
+#endif
+{
+    internal sealed class FormatterOptions
+    {
+        [Display(
+            ResourceType = typeof(OptionsDisplayStrings),
+            Description = nameof(OptionsDisplayStrings.DisplayAttributeDescription_ConsoleOptions_FormatterOptions_IncludeScopes))]
+        public bool IncludeScopes { get; set; }
```

Comment:
This probably has a default value?

Reply:
This doesn't imply that it does for IncludeScopes (unless I'm just looking at the wrong page): https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.console.consoleformatteroptions?view=dotnet-plat-ext-5.0